src/config.rs

```diff
@@ -29,6 +29,9 @@ pub(crate) const DEFAULT_ESPLORA_SERVER_URL: &str = "https://blockstream.info/ap
 // The timeout after which we abandon retrying failed payments.
 pub(crate) const LDK_PAYMENT_RETRY_TIMEOUT: Duration = Duration::from_secs(10);
 
+// The interval (in block height) after which we archive pruned channel monitors.
+pub(crate) const LDK_CHANNEL_MONITOR_ARCHIVAL_INTERVAL: u32 = 6;
```

Comment:
`chain_monitor::archive_fully_resolved_channel_monitors` needs `4032` blocks before archiving so not sure the docs are accurate.

Reply:
Yup, looks good